src/renderer/components/settings/field/password.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+    {{ field.display || field.name }}
+    <button @click="editing = !editing"></button>
+    <div v-if="editing">
+        <label>
+            <input
+                type="password"
+                :name="field.name"
+                :minLength="field.minLength"
+                :maxLength="field.maxLength"
+                :value="value"
+            />
+        </label>
+        <button @click="$emit('value-change', hashedJson(value))"></button>
+    </div>
+</template>
+
+<script>
+  import crypto from 'crypto'
+  
+  const SALT_LENGTH = 6
+
+  function hashedJson (password) {
+    let salt = crypto.randomBytes(Math.ceil(SALT_LENGTH/2))

```

Comment:
Your formula for the length is wrong. x bytes would represented by x*(4/3) in base64. meaning the right formula is `SALT_LENGTH / 4 * 3)`. But i think the better approach is just decide how much randomBytes would be randomized and take it's base64 would save you doing the `Math.ceil` and the `slice`.